node/test/@opent2t/test-b/TranslatorBC.js

```diff
@@ -1,78 +0,0 @@
-"use strict";
-// This code uses ES2015 syntax that requires at least Node.js v4.
-// For Node.js ES2015 support details, reference http://node.green/
-
-const EventEmitter = require("events");
-
-/**
- * This translator class implements two interfaces: "org.opent2t.test.B"
- * and "org.opent2t.test.C". Since those interfaces do not have any conflicting
- * member names, they are both implemented directly.
- */
-class TestTranslatorBC extends EventEmitter {
-
-    constructor(deviceProps) {
```

Comment:
@tjaffri If you agree, I shall merge.

---
In reply to: [75169591](https://github.com/openT2T/opent2t/pull/20#discussion_r75169591) [](ancestors = 75169591)